### PR TITLE
Added "Powitajka"

### PR DESCRIPTION
**Pull request by iJankowski**
Dodałem głupotkę z powitaniem.

### CODE REVIEW

File: Dodawanie i Odejmowanie.cs [hunk 1]
```diff
@@ -3,13 +3,18 @@
 Console.ForegroundColor = ConsoleColor.Magenta;
 Console.WriteLine("Cześć, tu iRa! <3");
 
-Console.ForegroundColor = ConsoleColor.Yellow;
+Console.ForegroundColor = ConsoleColor.Green;
```
Comment (kubaruczynski) [inline Dodawanie i Odejmowanie.cs:6]: IMHO this should stay yellow, we're using this everywhere. Let's keep consistency 🎆

File: Dodawanie i Odejmowanie.cs [hunk 2]
```diff
@@ -3,13 +3,18 @@
...
 Console.WriteLine("Witam w programie Kalkulator:");
 
 /* Zaczynamy się uczyć na seriou! Materiał szkoleniowy:
  1 readkey= odczytuje jeden znak z klawiatury 
  2 readline= odczytuje ciag do wcisniescia enter*/
- 
+
+Console.WriteLine("Hello World! What is your name?");
```
Comment (kubaruczynski) [inline Dodawanie i Odejmowanie.cs:13]: suggestion: maybe we can use `what's` to make it shorter?

File: Dodawanie i Odejmowanie.cs (hunk 1 above)
Comment (iJankowski) [reply Dodawanie i Odejmowanie.cs:6]: Our ceo said everywhere green from now @kubaruczynski 🤷‍♂️

File: Dodawanie i Odejmowanie.cs (hunk 2 above)
Comment (iJankowski) [reply Dodawanie i Odejmowanie.cs]: git gut @kubaruczynski

Comment (iJankowski) [reply Dodawanie i Odejmowanie.cs]: done.

**Review by kubaruczynski — APPROVED** [review]

**Pull request merged**